apps/web/components/Shell.tsx

```diff
@@ -342,7 +342,7 @@ const Layout = ({
                   "px-4 sm:px-6 md:px-8",
                   props.flexChildrenContainer && "flex flex-1 flex-col"
                 )}>
-                {props.children}
+                {!props.isLoading ? props.children : props.customLoader}
```

Comment:
Display custom loader instead of children - allows us to easily fit the layout of the page for custom components